etc/eslint/rules/typescript.js

```diff
@@ -0,0 +1,2426 @@
+*
+* @name typedef
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [typedef]{@link https://typescript-eslint.io/rules/typedef}
+*
+* @example
+* // Bad...
+* let bad;
+*
+* // Good...
+* let good: number;
+*/
+rules[ '@typescript-eslint/typedef' ] = 'error';
+
+/**
+* Avoids using an unbound method reference outside of a method’s body.
+*
+* @name unbound-method
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [unbound-method]{@link https://typescript-eslint.io/rules/unbound-method}
+*
+* @example
+* // Bad...
+* const bad = this.handleClick;
+*
+* // Good...
+* const good = this.handleClick.bind(this);
+*/
+rules[ '@typescript-eslint/unbound-method' ] = 'error';
+
+/**
+* Enforces that overloaded members must be consecutive.
+*
+* @name unified-signatures
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [unified-signatures]{@link https://typescript-eslint.io/rules/unified-signatures}
+*
+* @example
+* // Bad...
+* declare function bad( x: number ): number;
+* // Some unrelated overload
+* declare function bad( x: string ): string;
+* declare function bad( x: number | string ): number | string;
+*
+* // Good...
+* declare function good( x: number ): number;
+* declare function good( x: number | string ): number | string;
+* declare function good( x: string ): string;
+*/
+rules[ '@typescript-eslint/unified-signatures' ] = 'error';
+
+/**
+* Requires parentheses around arrow function arguments.
+*
+* @name arrow-parens
+* @memberof rules
+* @type {Array}
+* @default [ 'error', 'always' ]
+* @see [arrow-parens]{@link https://eslint.org/docs/rules/arrow-parens}
+*
+* @example
+* // Bad...
+* const bad = x => x * x;
+*
+* // Good...
+* const good = (x) => x * x;
+*/
+rules[ 'arrow-parens' ] = [ 'error', 'always' ];
+
+/**
+* Enforces one true brace style in block statements.
+*
+* @name brace-style
+* @memberof rules
+* @type {Array}
+* @default [ 'error', '1tbs' ]
+* @see [brace-style]{@link https://eslint.org/docs/rules/brace-style}
+*
+* @example
+* // Bad...
+* if (foo)
+* {
+*   bar();
+* }
+*
+* // Good...
+* if (foo) {
+*   bar();
+* }
+*/
+rules[ 'brace-style' ] = [ 'error', '1tbs' ];
+
+/**
+* Enforces that class methods utilize `this`.
+*
+* @name class-methods-use-this
+* @memberof rules
+* @type {string}
+* @default 'error'
+* @see [class-methods-use-this]{@link https://eslint.org/docs/rules/class-methods-use-this}
+*
+* @example
+* // Bad...
+* class MyClass {
+*   method() {
+*     console.log('no this');
+*   }
+* }
+*
+* // Good...
+* class MyClass {
+*   method() {
+*     console.log(this);
+*   }
+* }
+*/
+rules[ 'class-methods-use-this' ] = 'error';
+
+/**
+* Disallows trailing commas.
+*
+* @name comma-dangle
+* @memberof rules
+* @type {string}
+* @default 'off'
+* @see [comma-dangle]{@link https://eslint.org/docs/rules/comma-dangle}
+*
+* @example
+* // Okay...
+* const foo = {
+*   bar: 'baz',
+*   qux: 'quux'
+* };
+*/
+rules[ 'comma-dangle' ] = 'off';
```

Comment:
@Planeshifter Why is this rule disabled? Shouldn't this be `[ 'error', 'never' ]`?

Reply:
Changed.